main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name.strip(),
+            last_name=last_name.strip()
+        )
+
+    Actor.objects.filter(
```

Comment:
here would make sense to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..